MekHQ/src/mekhq/gui/StratconPanel.java

```diff
@@ -117,7 +117,10 @@
      * Constructs a StratconPanel instance, given a parent campaign GUI and a pointer to an info area.
      */
     public StratconPanel(CampaignGUI gui, JLabel infoArea) {
-        campaign = gui.getCampaign();
+        if (gui != null) {
```

Comment:
How does the rest of this survive without a `Campaign` instance? I see lots of other places where the code would die.

Reply:
To be honest, it's probably unnecessary. I wanted to re-test the StratconTab layout without having to reload the game, so I put in a call to InitTab in a couple of places, which re-initializes the StratconPanel - including once before the campaign GUI is loaded.

Reply:
Reverted.